src/webview/components/memory-widget.tsx

```diff
@@ -50,14 +54,27 @@ const defaultOptions: MemoryWidgetState = {
 };
 
 export class MemoryWidget extends React.Component<MemoryWidgetProps, MemoryWidgetState> {
+    protected optionsWidget = React.createRef<OptionsWidget>();
+    protected memoryTable = React.createRef<MemoryTable>();
     constructor(props: MemoryWidgetProps) {
         super(props);
         this.state = { ...defaultOptions };
     }
 
+    protected createVscodeContext(): VscodeContext {
+        const visibleColumns = `|${this.props.columns.filter(candidate => candidate.active).map(column => column.contribution.id).join('|')}|`;
```

Comment:
This was done to avoid partial/incorrect matching for columnids at the beginning/end of the string.
(`foobar|foo` ~= "bar|" would be true)
But maybe that was a bit of an unnecessary over generalization. In our case we know that all potential values and partial matching should not be an issue.

Also we could use a different separator (e.g. `;`) to avoid the need for escaping the `when` clauses